src/VisualStudio/Core/Def/Implementation/ProjectSystem/MiscellaneousFilesWorkspace.cs

```diff
@@ -30,7 +31,11 @@ internal sealed partial class MiscellaneousFilesWorkspace : Workspace, IVsRunnin
 
         private readonly Dictionary<Guid, LanguageInformation> _languageInformationByLanguageGuid = new Dictionary<Guid, LanguageInformation>();
 
-        private readonly HashSet<DocumentKey> _filesInProjects = new HashSet<DocumentKey>();
+        /// <summary>
+        /// <see cref="WorkspaceRegistration"/> instances for all open buffers being tracked by the workspace for possible
```

Comment:
"by this workspace" meaning this miscellaneousfilesworkspace? Maybe clearer without that phrase